examples/hello_milvus.ipynb

```diff
@@ -0,0 +1,359 @@
+      "hit: (distance: 0.3618723750114441, id: 1105), random field: -11.0\n",
+      "hit: (distance: 0.10607236623764038, id: 764), random field: -11.0\n",
+      "hit: (distance: 0.14412546157836914, id: 750), random field: -11.0\n",
+      "hit: (distance: 0.29973354935646057, id: 2716), random field: -11.0\n",
+      "search latency = 0.1434s\n"
+     ]
+    }
+   ],
+   "source": [
+    "hello_milvus.load()\n",
+    "\n",
+    "# search based on vector similarity\n",
+    "print(fmt.format(\"Start searching based on vector similarity\"))\n",
+    "vectors_to_search = entities[-1][-2:]\n",
+    "search_params = {\n",
+    "    \"metric_type\": \"l2\",\n",
+    "    \"params\": {\"nprobe\": 10},\n",
+    "}\n",
+    "\n",
+    "start_time = time.time()\n",
+    "result = hello_milvus.search(vectors_to_search, \"embeddings\", search_params, limit=3, output_fields=[\"random\"])\n",
+    "end_time = time.time()\n",
+    "\n",
+    "for hits in result:\n",
+    "    for hit in hits:\n",
+    "        print(f\"hit: {hit}, random field: {hit.entity.get('random')}\")\n",
+    "print(search_latency_fmt.format(end_time - start_time))\n",
+    "\n",
+    "# -----------------------------------------------------------------------------\n",
+    "# query based on scalar filtering(boolean, int, etc.)\n",
+    "print(fmt.format(\"Start querying with `random > -14`\"))\n",
+    "\n",
+    "start_time = time.time()\n",
+    "result = hello_milvus.query(expr=\"random > -14\", output_fields=[\"random\", \"embeddings\"])\n",
+    "end_time = time.time()\n",
+    "\n",
+    "print(f\"query result:\\n-{result[0]}\")\n",
+    "print(search_latency_fmt.format(end_time - start_time))\n",
+    "\n",
+    "# -----------------------------------------------------------------------------\n",
+    "# hybrid search\n",
+    "print(fmt.format(\"Start hybrid searching with `random > -12`\"))\n",
+    "\n",
+    "start_time = time.time()\n",
+    "result = hello_milvus.search(vectors_to_search, \"embeddings\", search_params, limit=3, expr=\"random > -12\", output_fields=[\"random\"])\n",
+    "end_time = time.time()\n",
+    "\n",
+    "for hits in result:\n",
+    "    for hit in hits:\n",
+    "        print(f\"hit: {hit}, random field: {hit.entity.get('random')}\")\n",
+    "print(search_latency_fmt.format(end_time - start_time))\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6. delete entities by PK\n",
+    "You can delete entities by their PK values using boolean expressions.\n",
+    "\n",
+    "Note: Right now, \"delete\" function is not implemented in the latest released version 2.0.0rc8, but it is supported in the lasted daily built version 2.0.0rc9dev27. which can be installed by `pip install --extra-index-url https://test.pypi.org/simple/ pymilvus==2.0.0rc9dev27`. Make sure you use a daily build version newer than v2.0.0-RC8 of Milvus accordingly (`docker pull milvusdb/daily-build:master-20211213-835e098`). "
```

Comment:
@DingQK Great that you made it work, but this note is unnecessary because the master branch is never aimed at users, it's for contributors. If one wants to contribute, he'll have to use the latest Milvus and PyMilvus.

Reply:
Hi @XuanYang-cn ! I delete the unnecessary note as requested.